Sources/HummingbirdWSClient/WebSocketClient.swift

```diff
@@ -186,14 +187,14 @@ extension WebSocketClient {
     ///   - eventLoopGroup: EventLoopGroup to run WebSocket client on
     ///   - logger: Logger
     ///   - process: Closure handling webSocket
-    public static func connect(
+    @discardableResult public static func connect(
```

Comment:
Let's document the return here e.g. when it's nil

Reply:
Ok good point